src/utils/update/psc.to.beneficial.owner.type.mapper.ts

```diff
@@ -0,0 +1,191 @@
+  OWNERSHIP_MORE_THAN_25_PERCENT_AS_FIRM = 'ownership-of-shares-more-than-25-percent-as-firm-registered-overseas-entity'
+}
+
+const natureTypeMap = new Map<string, string>([
+  [controlMap.OWNERSHIP_MORE_THAN_25_PERCENT_SHARE_ENTITY, BoTypes.BO_NATURE_OF_CONTROL],
+  [controlMap.VOTING_RIGHT_MORE_THAN_25_PERCENT_SHARE_ENTITY, BoTypes.BO_NATURE_OF_CONTROL],
+  [controlMap.RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS, BoTypes.BO_NATURE_OF_CONTROL],
+  [controlMap.SIGNIFICANT_INFLUENCE_OR_CONTROL_ENTITY, BoTypes.BO_NATURE_OF_CONTROL],
+  [controlMap.RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS_AS_TRUST, BoTypes.TRUST_NATURE_OF_CONTROL],
+  [controlMap.SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_TRUST, BoTypes.TRUST_NATURE_OF_CONTROL],
+  [controlMap.VOTING_RIGHT_MORE_THAN_25_PERCENT_AS_TRUST, BoTypes.TRUST_NATURE_OF_CONTROL],
+  [controlMap.OWNERSHIP_MORE_THAN_25_PERCENT_AS_TRUST, BoTypes.TRUST_NATURE_OF_CONTROL],
+  [controlMap.SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_FIRM, BoTypes.NON_LEGAL_NATURE_OF_CONTROL],
+  [controlMap.RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS_AS_FIRM, BoTypes.NON_LEGAL_NATURE_OF_CONTROL],
+  [controlMap.VOTING_RIGHT_MORE_THAN_25_PERCENT_AS_FIRM, BoTypes.NON_LEGAL_NATURE_OF_CONTROL],
+  [controlMap.OWNERSHIP_MORE_THAN_25_PERCENT_AS_FIRM, BoTypes.NON_LEGAL_NATURE_OF_CONTROL]
+]);
+
+const beneficialOwnerMap = new Map<string, string>([
+  [controlMap.OWNERSHIP_MORE_THAN_25_PERCENT_SHARE_ENTITY, NatureOfControlType.OVER_25_PERCENT_OF_SHARES],
+  [controlMap.VOTING_RIGHT_MORE_THAN_25_PERCENT_SHARE_ENTITY, NatureOfControlType.OVER_25_PERCENT_OF_VOTING_RIGHTS],
+  [controlMap.RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS, NatureOfControlType.APPOINT_OR_REMOVE_MAJORITY_BOARD_DIRECTORS],
+  [controlMap.SIGNIFICANT_INFLUENCE_OR_CONTROL_ENTITY, NatureOfControlType.SIGNIFICANT_INFLUENCE_OR_CONTROL],
+]);
+
+const trusteesNatureMap = new Map<string, string>([
+  [controlMap.OWNERSHIP_MORE_THAN_25_PERCENT_AS_TRUST, NatureOfControlType.OVER_25_PERCENT_OF_SHARES],
+  [controlMap.VOTING_RIGHT_MORE_THAN_25_PERCENT_AS_TRUST, NatureOfControlType.OVER_25_PERCENT_OF_VOTING_RIGHTS],
+  [controlMap.RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS_AS_TRUST, NatureOfControlType.APPOINT_OR_REMOVE_MAJORITY_BOARD_DIRECTORS],
+  [controlMap.SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_TRUST, NatureOfControlType.SIGNIFICANT_INFLUENCE_OR_CONTROL],
+]);
+
+const nonLegalFirmNatureMap = new Map<string, string>([
+  [controlMap.OWNERSHIP_MORE_THAN_25_PERCENT_AS_FIRM, NatureOfControlType.OVER_25_PERCENT_OF_SHARES],
+  [controlMap.VOTING_RIGHT_MORE_THAN_25_PERCENT_AS_FIRM, NatureOfControlType.OVER_25_PERCENT_OF_VOTING_RIGHTS],
+  [controlMap.RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS_AS_FIRM, NatureOfControlType.APPOINT_OR_REMOVE_MAJORITY_BOARD_DIRECTORS],
+  [controlMap.SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_FIRM, NatureOfControlType.SIGNIFICANT_INFLUENCE_OR_CONTROL]
+]);
+
+const mapDateOfBirth = (psc: CompanyPersonWithSignificantControl) => {
```

Comment:
Could have a general mapping function that takes psc.dataOfBirth (DateOfBirth) type. Need similar function for InputDate if mapping start_date